src/irmin-pack/unix/file_manager.ml

```diff
@@ -786,6 +835,11 @@ struct
         pl.chunk_start_idx pl.chunk_num];
     Control.set_payload t.control pl
 
+  let add_volume t =
```

Comment:
There are now two functions `add_volume` in this module. One at line 283 and this one. This one shadows the previous definition and is exposed.

Maybe we could improve by renaming the `add_volume` at 283 to something like `add_volume_to_lower_and_control_file`. That's very verbose, but something like that to distinguish it from the exported `add_volume` at 838.

Reply:
Good call. I'll find some name to prevent shadowing.